lib/service/connect.go

```diff
@@ -808,38 +808,43 @@ func (process *TeleportProcess) newClient(authServers []utils.NetAddr, identity
 
 	// Try and connect to the Auth Server. If the request fails, try and
 	// connect through a tunnel.
-	process.log.Debugf("Attempting to connect to Auth Server directly.")
+	logger := process.log.WithField("auth-addrs", utils.NetAddrsToStrings(authServers))
+	logger.Debugf("Attempting to connect to Auth Server directly.")
```

Comment:
You appear to be preferring plain `Debug` when there is nothing to substitute :)
```suggestion
	logger.Debug("Attempting to connect to Auth Server directly.")
```